### PR TITLE
Certain Welders can now be refueled while lit.

### PR DESCRIPTION
**Pull request by Mervill**
## About the PR 

**Changelog**

:cl:
- tweak: Certain Welders can now be refueled while lit.

### CODE REVIEW

**Comment by Mervill** [comment]
See also #11487

**Comment by EmoGarbage404** [comment]
probably only need the first line in your changelogs. Specifying the other individually is probably unnecessary.

File: Resources/Prototypes/Entities/Objects/Tools/welders.yml [hunk 1]
```diff
@@ -113,6 +115,8 @@
     enabled: false
     radius: 1.5
     color: lightblue
+  - type: Welder
+    tankSafe: true
```
Comment (moonheart08) [inline Resources/Prototypes/Entities/Objects/Tools/welders.yml:119]: Should not be tank safe.

File: Resources/Prototypes/Entities/Objects/Tools/welders.yml [hunk 2]
```diff
@@ -91,6 +91,8 @@
         maxVol: 250
   - type: Tool
     speed: 1.3
+  - type: Welder
+    tankSafe: true
```
Comment (moonheart08) [inline Resources/Prototypes/Entities/Objects/Tools/welders.yml:95]: Should not be tank safe.

**Review by moonheart08 — CHANGES_REQUESTED** [review]

**Comment by Elijahrane** [comment]
the issue you linked literally says the tankSafe field doesn't work. None of these should be tankSafe anyway. The issue is in the C# system where the refuel explosion doesn't check if the welder is tank safe.

**Comment by Mervill** [comment]
If none of the main-line welders are going to be tank safe, we should probably just remove tank safety.

**Comment by mirrorcult** [comment]
it seems reasonable for the advanced welder to be tank safe

**Pull request closed without merging**